### PR TITLE
Ridge CSI driver: update parameters

### PR DESCRIPTION
**Pull request by alex-ridge**

### CODE REVIEW

File: config-templates/ridge/ridge-csi-driver/1.0.0/custom-parameters.json [hunk 1]
```diff
@@ -29,23 +16,26 @@
     "description": "Ridge CSI image",
```
Comment (anandkarra) [inline config-templates/ridge/ridge-csi-driver/1.0.0/custom-parameters.json:16]: Can you add a better description for this? Something along the lines of "Location/URL of the Ridge CSI image registry".

File: config-templates/ridge/ridge-csi-driver/1.0.0/custom-parameters.json [hunk 2]
```diff
@@ -29,23 +16,26 @@
...
+    "mutable": "true"
   },
   {
-    "description": "Ridge CSI version",
-    "displayname": "Ridge CSI version",
-    "name": "ridge-csi-version",
-    "required": "true",
+    "description": "Ridge CSI Parameters (DaemonSet).",
+    "displayname": "Ridge CSI Parameters (DaemonSet).",
+    "name": "daemonset-parameters",
```
Comment (mssachan) [inline config-templates/ridge/ridge-csi-driver/1.0.0/custom-parameters.json:26]: Can u list all the options which can be passed to this param?

File: config-templates/ridge/ridge-csi-driver/1.0.0/custom-parameters.json [hunk 3]
```diff
@@ -29,23 +16,26 @@
...
+    "type": "csv",
     "mutable": "true"
   },
   {
-    "description": "Ridge CSI URL",
-    "displayname": "Ridge CSI URL",
-    "name": "ridge-csi-url",
+    "description": "Ridge CSI Parameters (StatefulSet)",
+    "displayname": "Ridge CSI Parameters (StatefulSet)",
+    "name": "statefulset-parameters",
```
Comment (mssachan) [inline config-templates/ridge/ridge-csi-driver/1.0.0/custom-parameters.json:35]: Can u list all the options which can be passed to this param?

**Comment by alex-ridge** [comment]
Too many conflicts, opened #270.

**Pull request closed without merging**